### DISCUSSION TITLE
Why Coin in Chia wallet is a lost.

### DISCUSSION THREAD

**Discussion by Wutchira** [Chia-Network/chia-blockchain#8113] [Support]
Hello.. CHIA SUPPORT

What happened the coin in Chia wallet application is LOST after restart computer?
(History transaction not have the Outgoing coin)
![228946363_4923932777621875_3621243270570782434_n](https://user-images.githubusercontent.com/89126576/129856773-f49f576b-4453-41d8-83e3-3049b63de008.jpg)
![229442054_4923932820955204_1059488958325748753_n](https://user-images.githubusercontent.com/89126576/129856777-a6cdb14f-d9a8-40a7-af33-6389ff6d5164.jpg)
![Capture](https://user-images.githubusercontent.com/89126576/129857378-f07cc34f-d057-4979-8206-b26e6dc56e4a.JPG)

**Comment by BrandtH22** [comment]
@Wutchira , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
